tests/tests/apihost.js

```diff
@@ -0,0 +1,94 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+"use strict";
+
+var utils = require('../utils');
+var actionSelector = utils.actionSelector;
+var appDetailsTitleSelector = utils.appDetailsTitleSelector;
+var very_short_wait = utils.very_short_wait;
+var short_wait = utils.short_wait;
+var medium_wait = utils.medium_wait;
+var very_long_wait = utils.very_long_wait;
+
+module.exports = {
+};
+
+module.exports['Install and launch test app'] = function (browser) {
+  browser
+    .init()
+    .installApp("https://alpha-qkhxczi7kki1x49pfakw.sandstorm.io/apihost-testapp.spk", "e148b304d9642e8c98ef8ea2df2f72ca", "w304h9n5rjx1pzfa8e4guheue5mq3dkwv63aajy1rscupw6e38mh")
```

Comment:
Done -- and extended the test to cover the DAV header, which I had forgotten about.